html/pages/device/showconfig.inc.php

```diff
@@ -64,26 +64,75 @@
                 $sep = ' | ';
             }
         }//end if
+        if ($config['rancid_repo_type'] == 'git') {
+            $sep     = ' | ';
+
+            chdir($configs);
+            $gitlogs_raw = external_exec('git log -n 8 --pretty=format:"%h;%ct" ' . $file);
```

Comment:
Can you convert the `external_exec()` calls to use `Process`, like: https://github.com/librenms/librenms/blob/master/app/Jobs/PingCheck.php#L86